docker/README.md

```diff
@@ -33,32 +33,20 @@ The following environment variables can be set with `-e` to control the test run
   PostgreSQL databases where the main store and state store are split.
 - `WORKERS`: set non-empty to test a worker-mode deployment rather than a
   monolith. Requires `POSTGRES`.
-- `REDIS`: set non-empty to use redis replication rather than old
-  TCP. Requires `WORKERS`.
 - `OFFLINE`: set non-empty to avoid updating the python or perl dependencies.
 - `BLACKLIST`: set non-empty to change the default blacklist file to the
   specified path relative to the Synapse directory
 - `TIMEOUT_FACTOR`: sets a number that test timeouts are multiplied by.
 
 Some examples of running Synapse in different configurations:
```

Comment:
Since there is now only one example here perhaps it makes sense to change this, i.e. "An example running Synapse in worker mode"? or similar?

Reply:
Good idea!